modules/tensor_mechanics/test/tests/lagrangian/updated/thermal_expansion/free.i

```diff
@@ -0,0 +1,278 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 2
+  ny = 2
+  nz = 2
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+  large_kinematics = false
+[]
+
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+  [disp_z]
+  []
+[]
+
+[AuxVariables]
+  [temperature]
+  []
+  [strain_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_xz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [strain_yz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xx]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_zz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_yz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [stress_xz]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[AuxKernels]
+  [control_temperature]
+    type = FunctionAux
+    variable = temperature
+    function = temperature_control
+  []
+
+  [stress_xx]
+    type = RankTwoAux
+    rank_two_tensor = cauchy_stress
```

Comment:
Looks like you added those to the `generate_outputs` parameter of the TensorMechanics Master action. Might as well use that functionality then.

Reply:
Yep, may as well.  Had to regold because the action calls it (correctly) `cauchy_stress`.